resources/public/css/orchestrator.css

```
@@ -992,8 +1225,9 @@ body {
     vertical-align: top;
 }
 
-#clusters_analysis .popover-content span.glyphicon {
-	font-family: "Glyphicons Halflings";
+#clusters_analysis .popover-content span.bi,
+#clusters_analysis .popover-body span.bi {
+	font-family: "bootstrap-icons";
```

Stylelint (17.14.0)

[error] Expected no quotes around "bootstrap-icons" (font-family-name-quotes)

(font-family-name-quotes)